modules/mobile/docs/examples/appointments/va_appointment_request.yml

```diff
@@ -24,6 +24,7 @@ data:
           extension: null
         url: null
         code: null
+      physicalLocation: null
```

Comment:
Does physical location only apply to VA appointments and not CC and requests?

Reply:
physical location only apply to VA appointments but I'm not sure if the would include va appointment requests.